compiler/front/commands.nim

```diff
@@ -7,19 +7,27 @@
 #    distribution, for details about the copyright.
 #
 
-# This module handles the parsing of command line arguments.
+## This module handles the parsing of command line arguments.
 
-# We do this here before the ``import`` statement so that defines from
-# inside the imported modules do not affect the boot switches. It's unlikely
-# to happen, but not impossible.
-template bootSwitch(name, expr, userString) =
-  # Helper to build boot constants, for debugging you can 'echo' the else part.
-  const name = if expr: " " & userString else: ""
 
-bootSwitch(usedRelease, defined(release), "-d:release")
-bootSwitch(usedDanger, defined(danger), "-d:danger")
-# `useLinenoise` deprecated in favor of `nimUseLinenoise`, kept for backward compatibility
-bootSwitch(useLinenoise, defined(nimUseLinenoise) or defined(useLinenoise), "-d:nimUseLinenoise")
+## Switches specified when the compiler is built (-d:xxx)
+## 
+## Don't use the constant for anything other than printing.
```

Comment:
ö

Reply:
Agreed, it does feel janky, a proper solution likely requires some marker to indicate the associativity of the comment and/or revise the comment association heruistic... but it works this way now so that's the constraint. 😬

I'm down for revising how comment association works (outisde this PR) so this jank can be avoided in the future.

Reply:
I fixed it now. I remembered fixing it.

Reply:
Thank you very much.